lib/pronto/golang/tools/base.rb

```diff
@@ -14,11 +14,11 @@ def initialize(config)
       end
 
       def command(file_path)
-        "#{base_command} #{parameters}"
+        "#{base_command} #{parameters} #{file_path}"
       end
 
       def parameters
-        @config.fetch('parameters', './...') # Default to './...' if the key is not configured
+        @config['parameters']
```

Comment:
`fetch` was used here on purpose as well.
If `./...` is not an option for a default anymore, that would have to be a blank string